server/uno_test.go

```diff
@@ -5,11 +5,15 @@ import (
 	"github.com/jak103/uno/db"
 	"github.com/jak103/uno/model"
 	"github.com/stretchr/testify/assert"
+	"os"
+	"errors"
 )
 
+
+
 // This function is meant to get a game and a player into the data base in a usable state for testing.
 func setupGameWithPlayer(database *db.DB) (*model.Game, *model.Player) {
-
+	os.Setenv("DB_TYPE", "MOCK")
```

Comment:
I really am confused... as this shouldn't be necessary. I'd love to review this after class if you have time.

Reply:
Haha I'm the one who was confused, there's a bit of new content here for me, so I had a little learning curve, but I think I understand now. Thanks for the help, here and in thew discord!